tools/ci_build/build.py

```diff
@@ -916,7 +919,8 @@ def generate_build_tree(
         "-Donnxruntime_USE_TENSORRT=" + ("ON" if args.use_tensorrt else "OFF"),
         "-Donnxruntime_SKIP_AND_PERFORM_FILTERED_TENSORRT_TESTS="
         + ("ON" if not args.tensorrt_placeholder_builder else "OFF"),
-        "-Donnxruntime_USE_TENSORRT_BUILTIN_PARSER=" + ("ON" if args.use_tensorrt_builtin_parser else "OFF"),
+        "-Donnxruntime_USE_TENSORRT_BUILTIN_PARSER="
```

Comment:
What if user manually specifies both --use_tensorrt_oss_parser and --use_tensorrt_builtin_parser? From the code it seems the build will not use built-in parser, right?